src/ResourceManager/DevTestLabs/Commands.DevTestLabs/Commands.DevTestLabs.csproj

```diff
@@ -101,27 +98,19 @@
     <Reference Include="Microsoft.Azure.Graph.RBAC">
       <HintPath>..\..\..\packages\Microsoft.Azure.Graph.RBAC.3.4.0-preview\lib\net452\Microsoft.Azure.Graph.RBAC.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Authorization">
```

Comment:
Because you *properly* removed this file, you will need to regenerate the msi.  Please follow the instructions here:  https://github.com/Azure/azure-powershell/wiki/Azure-Powershell-Developer-Guide#updating-the-installer for doing this.  This is causing a failure in the signing job